.github/workflows/smoke_test.yml

```
@@ -42,22 +42,16 @@ jobs:
           MIX_ENV=prod mix escript.build
           mv exercism_test_helper ../bin
 
-      - name: Smoke Test
-        run: |
-          ./bin/run.sh hello-world ./test/hello-world ./test/hello-world &&
-          ./bin/run.sh need-for-speed ./test/need-for-speed ./test/need-for-speed &&
-          ./bin/run.sh community-garden ./test/community-garden ./test/community-garden &&
-          ./bin/run.sh pythagorean-triplet ./test/pythagorean-triplet ./test/pythagorean-triplet &&
-          ./bin/run.sh bread-and-potions ./test/bread-and-potions ./test/bread-and-potions
-
-      - name: Verify files present
-        run: |
-          ./.github/bin/check_files.sh hello-world &&
-          ./.github/bin/check_files.sh need-for-speed &&
-          ./.github/bin/check_files.sh community-garden &&
-          ./.github/bin/check_files.sh pythagorean-triplet &&
-          ./.github/bin/check_files.sh bread-and-potions
-
       - name: Smoke tests for all exercises
         run: |
           ./bin/test_all_exercises.sh
+
+  smoke_test_in_docker:
```

```suggestion
  test_in_docker:
```

Agreed with @angelikatyborska, this isn't a smoke test anymore as it is running tests against all exercises (which is fine)

---

This one is actually a smoke test (running on about 5 exercises). It's the one on line 45 that should be renamed.